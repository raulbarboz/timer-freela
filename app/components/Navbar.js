import React from 'react';

function Navbar(){
    return(
        <div className="menu">
            <ul>
                <li>Freela</li>
            </ul>
        </div>
    )
}

export default Navbar;