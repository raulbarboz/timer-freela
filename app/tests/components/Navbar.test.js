import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Navbar from '../../components/Navbar';

test('should render Navbar correctly', () => {
    const wrapper = shallow(<Navbar />);
    expect(toJSON(wrapper)).toMatchSnapshot()
})