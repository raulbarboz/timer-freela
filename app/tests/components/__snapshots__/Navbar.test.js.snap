// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Navbar correctly 1`] = `
<div
  className="menu"
>
  <ul>
    <li>
      Telzir
    </li>
  </ul>
</div>
`;
